cfme/tests/configure/test_access_control.py

```diff
@@ -240,6 +277,34 @@ def test_group_crud():
     group.delete()
 
 
+@pytest.mark.tier(2)
+def test_group_crud_with_tag(a_provider, category, tag):
+    """Test for verifying group create with tag defined
+
+    Steps:
+        * Login as Admin user
+        * Navigate to add group page
+        * Fill all fields
+        * Set tag
+        * Save group
+    """
+    group = Group(
+        description='grp{}'.format(fauxfactory.gen_alphanumeric()),
+        role='EvmRole-approver',
+        tag=[category.display_name, tag.display_name],
+        host_cluster=[a_provider.data['name']],
+        vm_template=[a_provider.data['name'], a_provider.data['datacenters'][0],
+                     'Discovered virtual machine']
+    )
+    group.create()
+    with update(group):
+        group.tag = [tag.category.display_name, tag.display_name]
+        group.host_cluster = [a_provider.data['name']]
+        group.vm_template = [a_provider.data['name'], a_provider.data['datacenters'][0],
+                             'Discovered virtual machine']
+    group.delete()
```

Comment:
if any error happen during update for example, group will never be removed. 
I'd advice you to use either yield_fixture or request with group delete